com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.RenderGraph.cs

```diff
@@ -1644,6 +1644,34 @@ void RenderGizmos(RenderGraph renderGraph, HDCamera hdCamera, TextureHandle colo
             return executed;
         }
 
+        private class UpdatePostProcessScreenSizePassData
+        {
+            public int postProcessWidth;
+            public int postProcessHeight;
+            public HDCamera hdCamera;
+            public ShaderVariablesGlobal shaderVariablesGlobal;
+        }
+
+        internal void UpdatePostProcessScreenSize(RenderGraph renderGraph, HDCamera hdCamera, int postProcessWidth, int postProcessHeight)
+        {
+            using (var builder = renderGraph.AddRenderPass("Update RT Handle Scales CB", out UpdatePostProcessScreenSizePassData passData))
+            {
+                passData.hdCamera = hdCamera;
```

Comment:
In theory passes with no resource output are supposed to be culled by the graph. It has been disabled for now but I want to revisit this at some point. Since this pass has no output, you should add builder.AllowCulling(false) here to make it future prool :)